requirements.txt

```diff
@@ -1,79 +1,29 @@
 #
-# This file is autogenerated by pip-compile with Python 3.9
-# by the following command:
+# This file is autogenerated by pip-compile with python 3.9
+# To update, run:
 #
 #    pip-compile requirements.in
 #
```

Comment:
@sjahl adding you as reviewer mostly for a sanity check if anything here looks concerning/ if you think there is anything we might need to change in the docker image related to this

Reply:
Nope, I don't think anything with regards to the image needs to change. This looks reasonable to me -- we had quite a lot of transitive dependencies from hail!